asset_symlink.gemspec

```
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_development_dependency "appraisal"
-  spec.add_runtime_dependency "rails", '>=4', '<5.2'
+  spec.add_runtime_dependency "rails", '>=4', '<=5.2'
```

I think we should loosen this a bit more, for 5.2.x, as suggested.
```suggestion
  spec.add_runtime_dependency "rails", '>=4', '< 6.0'
```